Makefile

```diff
@@ -145,6 +145,29 @@ scoop: all
 	@ echo "Running minimal example with SCOOP"
 	python -m scoop -n 2 rmg.py -v testing/scoop/input.py
 
+.ONESHELL:
+update:
+	@ echo "Updating RMG-py"
+	@ echo "Please remember to stash unwanted changes."
+	git status
+	@ read -p "Are there any changes that are not staged for committed? [y|n] " user_input
+	if [ "$$user_input" = "N" ] || [ "$$user_input" = "n" ]; then
+		git checkout main || true
+		git pull git@github.com:ReactionMechanismGenerator/RMG-Py.git main || git pull https://github.com/ReactionMechanismGenerator/RMG-Py.git main || true
+		conda activate rmg_env
```

Comment:
Would be nice if the environment name weren't hard coded. Mine has a different name, and sometimes I have more than one. 
Perhaps instead have a check that RMG (or its dependencies) are available in the current environment, and a warning to activate the right environment if it is not.

You could probably just add `check` as a dependency for this target? (I think that checks all the python dependencies)